clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -94,6 +94,15 @@ class checker_no_abort_t : public invariant_checker_t {
         return true;
     }
 
+    // Pretend we skipped (this is far easier than adding a lot of logic to
+    // default_memtrace_stream_t and handling is_a_unit_test() in the checker).
```

Comment:
Remove stray `)` at the end.

Reply:
It is not stray as it matches `(this...` on the prior line.

Reply:
Oops. Glanced though. Sorry.